### PR TITLE
Update application.yml + remove mixer support.

### PR DESCRIPTION
**Pull request by Drapersniper**

### CODE REVIEW

File: redbot/cogs/audio/core/utilities/validation.py [hunk 1]
```diff
@@ -41,7 +41,6 @@ def is_url_allowed(self, url: str) -> bool:
             "bandcamp.com",
             "vimeo.com",
             "beam.pro",
```
Comment (Dessa) [inline redbot/cogs/audio/core/utilities/validation.py]: is there a reason to keep this one?

Comment (Drapersniper) [inline redbot/cogs/audio/core/utilities/validation.py]:
```suggestion
```

**Comment by aikaterna** [comment]
Why are you changing local playback to false?

File: redbot/cogs/audio/data/application.yml [hunk 2]
```diff
@@ -10,17 +10,39 @@ lavalink:
       soundcloud: true
       twitch: true
       vimeo: true
-      mixer: true
       http: true
-      local: true
-    sentryDsn: ""
-    bufferDurationMs: 1000
-    youtubePlaylistLoadLimit: 10000
+      local: false
```
Comment (Drapersniper) [inline redbot/cogs/audio/data/application.yml]:
```suggestion
      local: true
```

**Review by aikaterna — APPROVED** [review]

**Pull request merged**